chatkit-core/build.gradle

```diff
@@ -141,3 +142,11 @@ publishing {
         }
     }
 }
+
+detekt {
+    baseline = file("$projectDir/config/baseline.xml")
+
+    reports {
+        html.enabled = true
```

Comment:
How can we access those reports on Travis? I think it's crucial to have them accessible for the tool to provide value in a smooth way (not requiring re-runs on our machines).

Reply:
That's a good question, I'm not 100% sure how we access travis's files and I think that's kind of a problem in general with our usage of travis. 
Travis should tell us which task failed e.g. chatkit:core detekt and we can then establish what's happening ourselves locally (like we do when we get a failed test). Running `detekt` locally takes less than 3 seconds so it should not be too bothersome.